### DISCUSSION TITLE
Open a tzr file in Icepak AEDT

### DISCUSSION THREAD

**Discussion by mjmashayekhi** [ansys/pyaedt#4318] [General]
I'm trying to write a script to open a tzr project in Icepak AEDT. I wrote this script but it gives me error: 

from pyaedt import Icepak
import os

# Initialize Icepak object
ipk = Icepak()

# Set the TZR file path
tzr_file_path = "E:/GDDR7/Thermal Simulations - Samsung/20230724 - D1z_32Gb_GDDR7_XV_32Gbps_230724.tzr"

# Set the destination directory where you want to copy the project
destination_directory = "E:/GDDR7/Thermal Simulations - Samsung"

# Create the destination directory if it does not exist
if not os.path.exists(destination_directory):
    os.makedirs(destination_directory)

# Set the destination file path
destination_file_path = os.path.join(destination_directory, os.path.basename(tzr_file_path))

# Copy the project to the destination directory
ipk.copy_project(tzr_file_path, destination_file_path)

# Open the copied project
ipk.import_project(destination_file_path)

# Set the active design
design_name = "YourDesign"
ipk.set_active_design(design_name)

# Save the project
ipk.save_project()

# Close Icepak
ipk.close_project()

# Exit pyAEDT
ipk.close_desktop()

**Comment by Samuelopez-ansys** [comment]
@lorenzovecchietti

**Comment by lorenzovecchietti** [comment]
Hi @mjmashayekhi, I'm not sure what the problem might be here. If I try to open a tzr file, everything works fine using a "normal" approach:
```
import pyaedt

file = r"C:\Users\lvecchie\test_file.tzr"
ipk = pyaedt.Icepak(file)
```

If you want to save the converted project in another folder, just add a line like: `ipk.save_project(dest_file)`

Let me know if this works for you.


You might want to reload the file once you save it.

**Comment by Samuelopez-ansys** [comment]
Hi @mjmashayekhi any feedback?
